### PR TITLE
return errorResponseDto if ssn already has a signedMemberEntity

### PR DESCRIPTION
**Pull request by AlexSJ1990**

### CODE REVIEW

File: src/main/kotlin/com/hedvig/rapio/comparison/web/dto/ErrorResponseDTO.kt [hunk 1]
```diff
@@ -1,5 +1,12 @@
...
 
 data class ErrorResponse(
-        val message : String
-)
\ No newline at end of file
+        val errorCode: ErrorCodes,
+        val errorMessage: String
+)
+
+enum class ErrorCodes {
```
Comment (johantjellden) [inline src/main/kotlin/com/hedvig/rapio/comparison/web/dto/ErrorResponseDTO.kt]: Default values with jackson

File: src/main/kotlin/com/hedvig/rapio/comparison/web/dto/ErrorResponseDTO.kt [hunk 2]
```diff
@@ -1,5 +1,12 @@
 package com.hedvig.rapio.comparison.web.dto
```
Comment (johantjellden) [inline src/main/kotlin/com/hedvig/rapio/comparison/web/dto/ErrorResponseDTO.kt]: Move to ...underwriter.transport package

File: src/main/kotlin/com/hedvig/rapio/comparison/ComparisonQuoteController.kt [hunk 3]
```diff
@@ -11,9 +11,9 @@ import java.util.*
 import javax.validation.Valid
 
 
-fun notAccepted(error:String) = ResponseEntity.status(402).body(ErrorResponse(error))
+fun notAccepted(error:String) = ResponseEntity.status(402).body(ExternalErrorResponseDTO(error))
```
Comment (palmenhq) [inline src/main/kotlin/com/hedvig/rapio/comparison/ComparisonQuoteController.kt]: Again, i will complain about `402` even if it's not in the scope for this PR. First - I believe the "proper" response name would be `Not Acceptable` and second the response code for that is `406` :) Also 406 should be for content type negotiation, so here also i would say this is a `422 Unprocessable entity` 😬

Comment (AlexSJ1990) [reply src/main/kotlin/com/hedvig/rapio/comparison/ComparisonQuoteController.kt]: whoops thanks had missed this one - thought i had changed all to a 422

**Pull request merged**